package.json

```diff
@@ -7,6 +7,13 @@
   "bin": {
     "pino": "./pretty.js"
   },
+  "files" : [
+    "pino.js", 
+    "browser.js",
+    "pretty.js",
+    "LICENEE",
```

Comment:
oups. almost missed. I had a typo and it still included LISENSE?
So we don't need to mention it. npm granbs it anyway.